examples/functions/bluesky-post/document.json

```diff
@@ -0,0 +1,8 @@
+{
+  "_type": "post",
```

Comment:
the projection in the setup includes _id. It should still work, but probably best to match it so there's no surprises in production.